NineChronicles.Headless/GraphTypes/StandaloneMutation.cs

```diff
@@ -90,6 +90,54 @@ IConfiguration configuration
                 }
             );
 
+            // TODO deprecate stageTx and use this.
+            Field<NonNullGraphType<TxIdType>>(
+                name: "stageTxTxId",
+                description: "Add a new transaction to staging and return TxId",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "payload",
+                        Description = "The base64-encoded bytes for new transaction."
+                    }
+                ),
+                resolve: context =>
+                {
+                    try
+                    {
+                        byte[] bytes = Convert.FromBase64String(context.GetArgument<string>("payload"));
+                        Transaction<NCAction> tx = Transaction<NCAction>.Deserialize(bytes);
+                        NineChroniclesNodeService? service = standaloneContext.NineChroniclesNodeService;
+                        BlockChain<NCAction>? blockChain = service?.Swarm.BlockChain;
+
+                        if (blockChain is null)
+                        {
+                            throw new InvalidOperationException($"{nameof(blockChain)} is null.");
+                        }
+
+                        if (blockChain.Policy.DoesTransactionFollowsPolicy(tx, blockChain))
+                        {
+                            blockChain.StageTransaction(tx);
+
+                            if (service?.Swarm is { } swarm && swarm.Running)
+                            {
+                                swarm.BroadcastTxs(new[] { tx });
+                            }
+
+                            return tx.Id;
+                        }
+
+                        context.Errors.Add(new ExecutionError("The given transaction is invalid."));
+                    }
+                    catch (Exception e)
+                    {
+                        context.Errors.Add(new ExecutionError("An unexpected exception occurred.", e));
+                    }
+
+                    return null;
+                }
+            );
```

Comment:
Is there a particular reason for adding a new staging tx mutation and not just modifying `stageTx`? Is it just to leave past mutations as is?

I feel like the name `stageTxTxId` is alittle confusing but I'm not sure what would be a better name 🤔. `stageTxReturnTxId` seems more clearer but the naming feels awkward 😅. Any thoughts on this? @planetarium/9c-backend